keyboards/ergodox_ez/keymaps/rmw/config.h

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+Edited like hell 2017+ by Reed Williams <rmwphd@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+/* USB Device descriptor parameter */
+#define DESCRIPTION     QMK keyboard firmware for Ergodox EZ
```

Comment:
One morning drashna will wake up with the idea of adding `DESCRIPTION` to the keyboard-level `config.h`. That morning your keymap will break, unless

```suggestion
#ifndef DESCRIPTION
#define DESCRIPTION     QMK keyboard firmware for Ergodox EZ
#endif
```